src/tests/ftest/cart/util/cart_logparse.py

```diff
@@ -601,8 +596,7 @@ def _load_pids(self):
         for line in self._fd:
             fields = line.split(None, 8)
             index += 1
-            l_pid = None
-            if len(fields) < 6 or len(fields[0]) != 17:
+            if len(fields) < 6 or len(fields[0]) != 17 or fields[0][2] != '/':
```

Comment:
What does '/' correspond to here? Or rather, is there an example output for this file/line?

Reply:
This is just bringing it line with L575 above, example output would be any DAOS log file.